client/extensions/woocommerce/app/products/product-variation-types-form.js

```diff
@@ -79,7 +79,7 @@ export default class ProductVariationTypesForm extends Component {
 					onChange={ this.updateName }
 				/>
 				<TokenField
-					placeholder={ i18n.translate( 'Comma separate these' ) }
+					placeholder={ i18n.translate( 'Red, Green, Blue' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Blue & Red' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).